src/Components/Articles.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from "react"
+import { Link } from "react-router-dom"
+import { fetchArticles } from "../../utils"
+
+function Articles () {
+
+    const [articles, setArticles] = useState([])
+
+    useEffect(() => {
+        fetchArticles().then(({articles}) => {
+            setArticles(articles)
+        })
+    })
```

Comment:
At the moment you've got an infinite loop here and your site keeps fetching all the articles. You must include at least an empty dependency array in your useEffect() to avoid that.